src/ckeditor.jsx

```diff
@@ -30,9 +30,9 @@ class CKEditor extends React.Component {
     var node = React.findDOMNode(this.refs.ck);
     $(node).text(this.toHtml(content));
 
-    var imageBrowserUrl = window.location.href
-      + '&ckImageBrowser=on'
-      + '&siteUrl=' + encodeURIComponent(this.getSiteUrl());
+    var imageBrowserUrl = window.location.pathname + '/ckImageBrowser'
+      + window.location.search + (window.location.search ? '&' : '?')
```

Comment:
If you append `/ckImageBrowser` to the URL, you're already assuming there is no querystring, so you might as well append `?` directly.